### ISSUE TITLE
Send emails

### ISSUE THREAD

**Issue by pavel-the-best** [jjs-dev/jjs#361]
# Request

I want to have ability to send e-mails (with, maybe, two-factor authentication codes) using API.
# Motivation

This adds additional security. IMHO this is a must-have feature.

**Comment by MikailBag** [comment]
I don't think it is important feature, because it can be implemented outside of JJS.
